Resources/Prototypes/LootTables/mining_loot_table.yml

```diff
@@ -1,43 +1,47 @@
 - type: entityLootTable
   id: MiningLootTable
   entries:
+  - prob: 0.5 #null value, doesn't drop anything
+    orGroup: Asteroid
   - id: SteelOre1
```

Comment:
EntitySpawnCollection is uh... special code.
Eventually it just needs to be re-done in a way that allows mutually exclusive spawns to specify an overall probability.

Reply:
Yeah, it's pretty rough. This was the best solution that didn't require rewriting all of it. I don't think it's too bad.